maintaining.rst

```diff
@@ -0,0 +1,254 @@
+
+The Ansible community hopes that you will find that maintaining a collection is as rewarding for you as having the collection content is for the wider community.
+
+.. contents:: Topics
+
+Collection maintainership
+=========================
+
+What is the collection maintenance
+----------------------------------
+
+Ansible collections community can be logically divided into four major groups:
+
+  1. Steering committee members (make decisions in scope of all the collections included in Ansible package and related areas).
+  2. Maintainers.
+  3. Contributors.
+  4. Users.
+
+Maintainers are people who are responsible for collection maintenance.
+
+The collection maintenance consists of the regular activities listed in these guidelines.
+
+In particular, collection maintainers:
+
+  - :ref:`review <Reviewing>` and :ref:`commit <Committing>` changes made by other contributors,
+  - :ref:`backport <Backporting>` changes to stable branches,
+  - address issues discovered to appropriate contributors,
+  - :ref:`release <Releasing>` collections,
+  - watch that collections adhear the `Collection Requirements <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst>`_,
+  - keep tracking changes announced in ``Changes impacting collection contributors and maintainers`` `GitHub issue <https://github.com/ansible-collections/overview/issues/45>`_ and timely update a collection in accordance with them.
+
+Who is a collection maintainer
+------------------------------
+
+A collection maintainer is a contributor trusted by the community who makes significant and regular contributions to the project and showed themselves as a specialist in the related area.
+
+Collection maintainers has :ref:`extended permissions <Collection maintainers>` in the collection scope.
+
+Collection maintainers satisfy the :ref:`requirements <Requirements for maintainers>`.
+
+Requirements for maintainers
+----------------------------
+
+Maintainers act in accordance with `Ansible Code of Conduct <https://docs.ansible.com/ansible/latest/community/code_of_conduct.html>`_.
+
+Maintainers (including candidates) have:
+
+  - History of multiple contribution to a collection.
+  - Excellent technical judgement in collection-related areas.
+  - Responsiveness to mentions in issues and pull requests.
+  - Responsiveness to issues and pull requests assigned to them.
+  - Read these guidelines and the linked documents.
+  - Subscribed to:
+
+    + a collection repository (the ``Watch`` button => ``All activity``),
+    + the ``Changes impacting collection contributors and maintainers`` `GitHub issue <https://github.com/ansible-collections/overview/issues/45>`_,
+    + the `Bullhorn newsletter <https://github.com/ansible/community/issues/546>`_.
+  - Knowledge and intention to manage a collection performing the tasks listed in these guidelines. Maintainers can divided responsibilities between each other.
```

Comment:
```suggestion
  - Knowledge and intention to manage a collection performing the tasks listed in these guidelines. Maintainers can divide responsibilities between each other.
```

Reply:
@felixfontein thanks, committed the both